src/scripts/types/ApplicationTypes.js

```diff
@@ -30,7 +30,7 @@ export type Action<T> = {
 }
 
 export type RootState = {
-  video: ?VideoRecord,
+  selectedVideo: ?VideoRecord,
```

Comment:
We might want to refine our state a bit more. What does `selectedVideo` mean, exactly? Is this going to be separate to a video you select w/ the purposes of just playing the video? Or is this only a video you are editing